homework9/src/siit/hw9/App.java

```diff
@@ -0,0 +1,59 @@
+package siit.hw9;
+
+import java.io.*;
+import java.time.Month;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class App {
+
+    public static void getPersonByMonth(String input, int month, String output) {
+
+        List<Person> list = new ArrayList<>();
+        try {
+            BufferedReader inputStream1 = new BufferedReader(new FileReader(input));
+
+
+            String fileLine;
+
+            // We read the  file and create Person object with the data within
+            while ((fileLine = inputStream1.readLine()) != null) {
+                String[] data = fileLine.split(",");
+                String e1 = data[0];
+                String e2 = data[1];
+                String e3 = data[2];
```

Comment:
It works assuming that the input file has the expected format; if it doesn't, the app crashes, possibly with an `ArrayIndexOutOfBoundsException`. Think about ways to improve this and program defensively (make it bullet proof).